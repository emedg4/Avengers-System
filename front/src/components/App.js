import React, { Component } from 'react';
import Wallet from './Wallet';
class App extends Component {
    render() {
        return (
            <div>
                <h2>Marvel Hero List</h2>
                <h4>
                    <Wallet />
                </h4>
            </div>
        )
    }
}

export default App;