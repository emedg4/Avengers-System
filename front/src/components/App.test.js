import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import App from './App';
Enzyme.configure({ adapter: new Adapter() });

describe('App', () => {
    const app = Enzyme.shallow(<App />);
    it('render properly', () => {
        expect(app).toMatchSnapshot();
    })

    it('contains a connceted Wallet component', () => {
        expect(app.find('Connect(Wallet)').exists()).toBe(true);
    });
});