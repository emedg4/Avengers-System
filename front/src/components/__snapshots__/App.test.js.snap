// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render properly 1`] = `ShallowWrapper {}`;
